.github/workflows/deploy.yml

```diff
@@ -14,7 +14,7 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This workflow contains a single job called "build"
```

Comment:
Nit: Update or remove the comment

Reply:
just removed the comment, `%help` actually isn't defined

Reply:
See, bot works - I just sent you 10 testnet near with it: https://explorer.testnet.near.org/transactions/A6tG27NhDU2DHQG3CKUj5spprMSZeNVaaHGtkxmUwrum